orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/HttpClientUtils.java

```diff
@@ -77,6 +78,13 @@ public long getRetryInterval() {
       }
     );
 
+    httpClientBuilder.setRetryHandler((exception, executionCount, context) -> {
```

Comment:
The exception here is an IOException. Also adding a wait otherwise, it will retry right away